### ISSUE TITLE
Want to launch bleachit_console automatically

### ISSUE THREAD

**Issue by BYTEMEE1** [bleachbit/bleachbit#1731]
To clean all the categories previously checked in the GUI, plus the added custom directories. The batch file, below, doesn't work. Trying to see if this is possible by other means.
                      
@echo off
"bleachbit_console.exe" --clean system.tmp system.logs system.cache
timeout /t 5
exit

**Comment by strotee** [comment]
I have a batch for this purpose and in it contains this line. It assumes BleachBit is installed at the default location. Use it at your discretion:

"C:\Program Files (x86)\BleachBit\bleachbit_console.exe" -o -c --preset

**Comment by az0** [comment]
> I have a batch for this purpose and in it contains this line. It assumes BleachBit is installed at the default location. Use it at your discretion:
> 
> "C:\Program Files (x86)\BleachBit\bleachbit_console.exe" -o -c --preset

Yes, that looks right. You figured it out, so I am closing the issue.
